client/src/actions/ConfigurationsActions.ts

```diff
@@ -117,6 +119,31 @@ class ConfigurationsActions extends AbstractActions implements IConfigurationsAc
     };
   }
 
+  saveAsTemplate(template: IDashboardConfig) {
+    
+    return (dispatcher: (result: { template: IDashboardConfig }) => void) => {
+      let script = this.objectToString(template);
+      
+      script = '/// <reference path="../../../client/@types/types.d.ts"/>\n' +
+              'import * as _ from \'lodash\';\n\n' +
+              'export const config: IDashboardConfig = /*return*/ ' + script;
```

Comment:
I meant that the 'return' word is commented out. if this is on purpose, and as i understand from you its for 'indication' than I guess a comment is better..